### PR TITLE
Add component definition for Jackson.

### LINKED ISSUE #7812: [Feature] Add Jackson plugin
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

Jackson has been known as "the Java JSON library" or "the best JSON parser for Java". Or simply as "JSON for Java". It's the default JSON serialization and deserialization tool in the Spring Framework.

### Use case

When the application service uses Jackson, it tracks the json serialization and deserialization(optional plug-in).

### Related issues

Nothing.

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by VictorZeng**
- [x] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #7812 .
- [x] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/CHANGES.md).

### CODE REVIEW

**Comment by wu-sheng** [comment]
Pending on https://github.com/apache/skywalking-java/pull/39

**Comment by VictorZeng** [comment]
Maybe the plugin should be in version 8.8.0.

File: changes/changes-8.8.0.md [hunk 1]
```diff
@@ -75,7 +75,7 @@ Release Notes.
   low down metrics flush period added in 8.7.0.
 * Fix `Slow SQL sampling` not accurate, due to TopN works conflict with cache read-then-clear mechanism.
 * The persistent cache is only read when necessary.
-* Add component definition for `Alibaba Fastjson`.
+* Add component definition for `Alibaba Fastjson` and `Jackson`.
```
Comment (wu-sheng) [inline changes/changes-8.8.0.md:78]: You should not move to here. The original version is correct.

Comment (wu-sheng) [reply changes/changes-8.8.0.md:78]: This is main repository's release log, which has started. https://lists.apache.org/thread.html/rde5ccf89acdf60215b4409097273bf171b81fdda8d22370f4a33414a%40%3Cdev.skywalking.apache.org%3E

Comment (wu-sheng) [reply changes/changes-8.8.0.md:78]: All change logs in `changes` folder are archived versions, unless there are typo, they should not be changed.

Comment (VictorZeng) [reply changes/changes-8.8.0.md:78]: OK, will reset it , but the java-agent verison is 8.8.0-SNAPSHOT, is OK?

Comment (wu-sheng) [reply changes/changes-8.8.0.md]: Java agent is going to use its own version from this month. There are no relationship, version relationship is described in https://skywalking.apache.org/docs/main/latest/en/setup/service-agent/agent-compatibility/

Comment (wu-sheng) [reply changes/changes-8.8.0.md]: Read this, if you want to see what is happening. https://github.com/apache/skywalking/discussions/7548

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**